flake.nix

```diff
@@ -8,7 +8,7 @@
     {
       packages = {
         x86_64-linux = rec {
-          nilla = project.packages.nilla-cli.result.x86_64-linux;
+          nilla-cli = project.packages.nilla-cli.result.x86_64-linux;
           default = nilla;
```

Comment:
We will need to update default too, right?

Reply:
yep, good catch, thanks!

Reply:
done